libs/application/template-api-modules/src/lib/modules/shared/api/user-profile/user-profile.service.ts

```diff
@@ -47,6 +53,24 @@ export class UserProfileService extends BaseTemplateApiService {
             400,
           )
         }
+
+        if (params?.validateEmail && !results?.email) {
+          throw new TemplateApiError(
+            {
+              title: coreErrorMessages.noEmailFound,
+              summary: {
+                ...coreErrorMessages.noEmailFoundDescription,
+                defaultMessage:
```

Comment:
You are just working with the default message. If summary has a value in contentful this replace wont happen. You will want to pass it in with the  `values`  property. 
```
          summary: {
            ...coreErrorMessages.noEmailFoundDescription,,
            values: { link: this.getIDSLink(application) },
          },
```